grblas/base.py

```diff
@@ -5,11 +5,51 @@
 
 NULL = ffi.NULL
 
-# Sentinel object to indicate GrB_OUTP = GrB_REPLACE in the descriptor
-REPLACE = object()
+
+class Updater:
+    def __init__(self, parent, **kwargs):
```

Comment:
Can we expose `mask=`, `accum=`, and `replace=` keywords here?  Also, what do you think about exposing these as attributes on the `Updater`?

Reply:
The Updater is not supposed to be part of the user-facing API, which is why we don't expose it in grblas/__init__.py.

If exposing the keywords makes the code clearer, I'm okay with this change, but I don't want to encourage normal users to directly start calling the Updater.

Reply:
`Updater` is an object people operate on.  It is not strictly behind-the-scenes.  It is public, and people will inspect it.  I agree users shouldn't get in the habit of calling `Updater` directly.  However, it could be fair for users to do `isinstance(x, Updater)`.